docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -98,7 +90,6 @@ Mbed CLI supports a setting for each toolchain path:
 
 | Toolchain | Setting name | Example binary location | Example path |
 | --------- | --------- | ---------| --------- |
-| Arm Compiler 5.06u6 | `ARM_PATH` | `C:\Program Files\ARM_Compiler_5.06u6\bin\armcc` | `C:\Program Files\ARM_Compiler_5.06u6` |
 | Arm Compiler 6.11 | `ARMC6_PATH` | `C:\Program Files\ARMCompiler6.11\bin\armclang` | `C:\Program Files\ARMCompiler6.11\bin` |
```

Comment:
We should update the version of AC6 to Arm Compiler 6.13, as it's the one supported and tested in CI and Mbed Studio.

Reply:
Yes, Hugue and I have already agreed on that.